### DISCUSSION TITLE
Moving from express

### DISCUSSION THREAD

**Discussion by 7heMech** [kartikk221/hyper-express#149] [Q&A] [answered]
While I was converting an express app to hyper express I got an error `express.engine is not a function` so I was wondering what is the equivalent to this
```js
app.engine('.html', require('ejs').renderFile);
app.set('view engine', 'html');
```
in hyper express?

**Comment by kartikk221** [comment, marked as answer]
HyperExpress is not 100% compatible with Express.js and at the moment engines are not supported by HyperExpress. You can however use middlewares to inject your own custom methods or use a middleware based solution for rendering content.
